test/models/user_test.rb

```diff
@@ -1,7 +1,12 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  context 'user basic info' do
+    setup do
```

Comment:
yeah this is a mistake on my part something I started but didnt finish. The user_test should be Parastoo's